web/controllers/homepage_controller.ex

```diff
@@ -138,7 +138,7 @@ defmodule App.HomepageController do
     @http.post_spreadsheet(data, url, type)
 
     conn
-      |> put_flash(:info, "#{String.capitalize(type)} collected")
+      |> put_flash(String.to_atom(type), "#{String.capitalize(type)} collected")
```

Comment:
I think to_atom method can be risky in certain case when the string can't be convert to an atom but on this case it will be ok :+1: